sig-windows/annual-report-2022.md

```diff
@@ -4,17 +4,18 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - [HostProcess containers](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/README.md) graduated to `stable` in v1.26
+   - Windows performance tests are not reported on the [k8s perf dashboard](https://perf-dash.k8s.io/#/?jobname=soak-tests-capz-windows-2019&metriccategoryname=E2E&metricname=CPUUsage)
 
```

Comment:
Guide for adding Windows nodes using kubeadm with flannel and calico CNIs? https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/guides/guide-for-adding-windows-node.md

Reply:
Prometheus Windows exporter for Kubernetes? https://github.com/prometheus-community/windows_exporter/pull/864

Reply:
Yes, let me add both of those!

Reply:
Updated!

Reply:
are you missing the commit with these? I don't seem them...

Reply:
Yes, i forgot to push :/
Should be there now!